subworkflows/nf-core/bcl_demultiplex/main.nf

```diff
@@ -8,12 +8,13 @@ include { BCLCONVERT } from "../../../modules/nf-core/bclconvert/main"
 include { BCL2FASTQ  } from "../../../modules/nf-core/bcl2fastq/main"
 
 // Define the log file path before the workflow starts
-def logFile = new File("${params.outdir}/invalid_fastqs.log")
+def logFile = new File("${params.outdir}/skipped_fastqs.log")
```

Comment:
I think we generally try to avoid using `params` in subworkflows 😦

Reply:
oh, if not it was ending in some temp workdir, this way it is saved inside the outdir provided in the `nextflow run --outdir /some/path/outdir`, shall I remove it and leave it as `def logFile = new File("skipped_fastqs.log")`?